test/ui-test/src/screens/StatusChatScreen.py

```diff
@@ -55,7 +57,17 @@ def send_message(self, message: str):
         type(ChatComponents.MESSAGE_INPUT.value, message)
         press_enter(ChatComponents.MESSAGE_INPUT.value)
         
-    # Verifications region:        
+    def clear_history(self):
+        click_obj_by_name(ChatComponents.MORE_OPTIONS_BUTTON.value)
+        click_obj_by_name(ChatComponents.CLEAR_HISTORY_MENUITEM.value)
+            
+    # Verifications region:      
+    def verify_last_message_is_not_loaded(self):
+        [loaded, _] = is_loaded_visible_and_enabled(ChatComponents.LAST_MESSAGE_TEXT.value)
+        if not loaded:
+            test.passes("Success: No message was found")
```

Comment:
Use `StatusVerificationDriver` insted. We don't use direct squish api calls in screens.